### DISCUSSION TITLE
Istanza zigbee2mqtt esterna (su PC WIN10) non comunica con Mosquitto broker ( su HASSIO)

### DISCUSSION THREAD

**Discussion by faustozago** [indomus/forum#2644] [ZigBee, Z-Wave, Thread, Matter]
Ciao a tutti,
uso Hassio su raspberry , dove ho gia' gli addon MOSQUITTO BROKER e ZIBBEE2MQTT funzionanti da tempo.
Ora ho installato una nuova istanza zigbee2mqtt su un pc con win10 , (seguendo la guida del sito ufficiale Z2M).
Sul PC funziona tutto, ho installato anche un dispositivo, funziona anche la connessione al frontend in remoto (dall'indirizzo http://192.168.x.xxx:8080.
Il problema è che non riesco a vedere le entita' di questa nuova istanza su HASSIO.

Quando lancio la nuova entita' su win, dal prompt, mi dice questo, a me sembra tutto OK:

> Zigbee2MQTT:info  2023-09-28 10:31:09: Connecting to MQTT server at mqtt://192.168.x.xxx
> Zigbee2MQTT:info  2023-09-28 10:31:10: Connected to MQTT server
> Zigbee2MQTT:info  2023-09-28 10:31:10: MQTT publish: topic 'zigbee2mqtt/bridge/state', payload 'online'

Qualcuno puo' aiutarmi? grazie.

**Comment by KC-inDomus** [comment]
gli hai cambiato il topic? all'istanza, intendo

**Reply by faustozago** [reply]
Scusami ma non sono espertissimo....intendi   " base_topic: zigbee2mqtt "  ?
Devo dargli un nome diverso ,visto che esiste gia' l'istanza di hassio con quel nome?

**Reply by KC-inDomus** [reply]
esatto

**Reply by faustozago** [reply]
Fatto, riavviato tutto ma ancora niente.
Dal frontend del nuovo Z2M ci sono delle configurazioni da fare?  per esempio:
-Impostazioni/integrazioni con Home assistant --------devo selezionare simpled o advanced?  Io non ho selezionato niente
-Impostazioni/ Mqtt-------?????
-Impostazioni/disponibilita' --------devo selezionare simpled o advanced?  Io non ho selezionato niente

**Reply by KC-inDomus** [reply]
integrazioni con home assistant dev'essere attivo. mqtt deve parlare col broker che di sicuro avrai già attivato.

**Reply by faustozago** [reply]
Ho spuntato "integrazione con home ass."  , ora sembra che funzioni ...

GRAZIE MILLE !!!

**Reply by KC-inDomus** [reply]
de nada
